docs/tips/15-expose-component-functions.zh-CN.md

```diff
@@ -0,0 +1,60 @@
+---
+id: expose-component-functions
+title: 暴露组件函数
+layout: tips
+permalink: tips/expose-component-functions.html
+prev: communicate-between-components.html
+next: children-undefined.html
+---
```

Comment:
Could you also change the above lines to
```
id: expose-component-functions-zh-CN
title: 暴露组件函数
layout: tips
permalink: tips/expose-component-functions-zh-CN.html
prev: communicate-between-components-zh-CN.html
next: children-undefined-zh-CN.html
```

We'll need the `-zh-CN` suffix for our i18n site to work in the future. Thanks! :)